site/docs/reference/Connectors/materialization-connectors/starburst.md

```diff
@@ -0,0 +1,136 @@
+# Starburst
+
+This connector materializes transactionally Flow collections into Iceberg or Delta Lake tables in [Starburst Galaxy](https://www.starburst.io/platform/starburst-galaxy/).
```

Comment:
Add also reference here about AWS S3.

The materialization connector don't support (yet)  other kind of object storages to persist to.